src/libraries/Microsoft.PowerFx.Core/Public/Config/Features.cs

```diff
@@ -45,6 +45,12 @@ public enum Features : int
         /// </summary>
         StronglyTypedBuiltinEnums = 0x10,
 
+        /// <summary>
+        /// Updates the IsEmpty function to only allow table arguments, since it
+        /// does not work properly with other types of arguments.
+        /// </summary>
+        RestrictedIsEmptyArguments = 0x20,
```

Comment:
This change is already in main, it should not be shown in the diff for your PR; maybe you need to rebase / merge from the latest main branch?